### ISSUE TITLE
add ignore list 

### ISSUE THREAD

**Issue by lqzhgood** [hackjutsu/Lepton#336]
#### Environment info
* OS: Windows
* App Version : 1.7

#### feature request
Add ignore list~
My Gist is not all of the code snippet. some config file in the Gist.  like Atom sync setting files.  they are add a lot of worry language tags in my Lepton.

So Can you add ignore list ?

I think ignorelist is a special and unique title in  Gist，can help Lepton ignore himself in content.
the ignorelist content is gist id. and have a page in Lepton setting to CRUD this ignorelist.

**Comment by onetrev** [comment]
Agreed this would be extremely helpful! I have a few gists like my VS Code settings sync and a few gists that I've forked over the years that I wanted to save or play around with, but definitely are not something I need to see in Lepton.

**Comment by SevnChen** [comment]
It is September 2021. Is there any way to implement the ignore list?
